test/Microsoft.ML.TestFrameworkCommon/TestCommon.cs

```diff
@@ -0,0 +1,285 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using Microsoft.ML.Data;
+using Xunit;
+
+namespace Microsoft.ML.TestFrameworkCommon
+{
+    public static class TestCommon
+    {
+        public static string GetOutputPath(string name, string outDir)
```

Comment:
This seems backwards to me. Shouldn't `outDir` go before `name`? Just like how `Path.Combine(outDir, name)` does?